### DISCUSSION TITLE
How Does Sign In With Farcaster Work?

### DISCUSSION THREAD

**Discussion by codexbushi** [wevm/frog-archived#236] [Q&A]
I am trying to get Sign In With Farcaster to work when using the Devtools. I am using Android so when I scan the QR code, it isn't deep linking to my Warpcast.

I changed the URL to be: https://client.warpcast.com/deeplinks/signed-key-request?token={TOKEN} 

This seems to work, it opens up my Warpcast and I paid the 10 Warps to connect. Now it will say I am connected, but my `Verified` is still `false` and I get this error:

```
Error: message is invalid. invalid signer: signer
```

Seems like I am missing a step, can someone help me out?

**Comment by dalechyn** [comment]
Several efforts were made to make this experience smoother.
Let us know if you have any problems.
